app/assets/stylesheets/wpcc/section/_details.scss

```diff
@@ -13,6 +13,11 @@
   }
 }
 
+.details__error-summary,
+.details__row--group {
+  @include column(12);
+}
+
```

Comment:
Is this to style the error summary and the message on the field? If so, do we need to do this lines in every project? If so are you considering add to the documentation that you're written?